_sass/_type.scss

```diff
@@ -4,32 +4,32 @@
     text-wrap: pretty;
   }
 
-  a,
-  a:link,
-  a:visited {
+  :where(a, a:link, a:visited) {
+    --_bg-progress: var(--link-bg-progress, 0);
+    --_bg-color: var(--link-bg-color, #{$orange});
+
     color: inherit;
     text-decoration: none;
     position: relative;
     font-weight: 700;
-    display: inline-block;
-
-    &::before {
-      content: '';
-      background-color: rgba(251, 147, 6, 0.75);
-      position: absolute;
-      left: 0;
-      bottom: 3px;
-      width: 100%;
-      height: 5px;
-      z-index: -1;
-      transition: all .3s ease-in-out;
+
+    padding-inline: 0.25rem;
```

Comment:
I'm not sure we need the left and right padding, especially now that we are animating up. It makes links in lists look too much indented.